manifests/e/Evernote/Evernote/10.101.6/Evernote.Evernote.installer.yaml

```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.4.1 Dumplings Mod $debug=QUSU.CRLF.7-4-4.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Evernote.Evernote
+PackageVersion: 10.101.6
+InstallerType: nullsoft
+Scope: user

```

Comment:
```suggestion
Scope: machine
```

Installation appears to be machine wide

![image](https://github.com/user-attachments/assets/388c0547-579a-43b4-9e2c-6d93268c16c9)

Reply:
Both scopes available. When the installer is run under admin, the machine-wide option will show up and become the default.